backend/infrahub/graphql/mutations/computed_attribute.py

```diff
@@ -46,6 +47,28 @@ async def mutate(
         ):
             raise ValidationError(input_value=f"{node_schema.kind}.{target_attribute.name} is not a computed attribute")
 
+        required_decision = PermissionDecision.ALLOW_OTHER
+        if context.branch.name == registry.default_branch:
+            required_decision = PermissionDecision.ALLOW_DEFAULT
+
+        has_update_permission = False
+        for permission_backend in registry.permission_backends:
+            if has_update_permission := await permission_backend.has_permission(
+                db=context.db,
+                account_session=context.active_account_session,
+                permission=ObjectPermission(
+                    namespace=node_schema.namespace,
+                    name=node_schema.name,
+                    action=PermissionAction.UPDATE.value,
+                    decision=required_decision.value,
```

Comment:
@gmazoyer, does the required_decision variable look as expected for you here? Have we managed this in another way elsewhere? Basically we know the namespace, name and branch of a node that we want to update.

Reply:
This seems to be alright with the current codebase. We have the same kind of logic in other places like:
```python
        required_decision = (
            PermissionDecisionFlag.ALLOW_DEFAULT
            if analyzed_query.branch is None
            or analyzed_query.branch.name in (GLOBAL_BRANCH_NAME, registry.default_branch)
            else PermissionDecisionFlag.ALLOW_OTHER
        )
```